src/main/java/org/opensearch/flowframework/exception/FlowFrameworkException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.exception;
+
+import javax.ws.rs.core.Response;
```

Comment:
Let's use the same status type as we'll be using in the Rest Handlers.
```suggestion
import org.opensearch.core.rest.RestStatus;
```